barretenberg/ts/src/barretenberg_api/index.ts

```diff
@@ -12,26 +12,6 @@ import {
   OutputType,
 } from '../serialize/index.js';
 import { Fr, Fq, Point, Buffer32, Buffer128, Ptr } from '../types/index.js';
```

Comment:
everyone was editing the file that said DO NOT EDIT. I have finally regenerated it. Luckily mostly people kept to the pattern